include/derivs.hxx.in.description

```diff
@@ -0,0 +1,16 @@
+/*!
```

Comment:
Is there any value in keeping a consistent file extension for the jinja templates?

Reply:
I have 2 jinja templates ... I didn't wanted toooo long file names ...